packages/components/src/internal/components/domainproperties/issues/IssuesListDefPropertiesPanel.tsx

```diff
@@ -93,6 +94,19 @@ export class IssuesListDefPropertiesPanelImpl extends React.PureComponent<
         }
     };
 
+    onCheckChange = e => {
```

Comment:
minor: this method makes it sound general and that it would apply to any checkbox change, but this is specific to a single checkbox. I'd suggest naming this something like `onRestrictedListCheckChange` or something like that.